bottery/message.py

```diff
@@ -14,17 +14,19 @@ class Message:
     text = attr.ib()
     timestamp = attr.ib()
     raw = attr.ib()
+    templates_dirs = attr.ib()
 
     @property
     def datetime(self):
         return datetime.utcfromtimestamp(self.timestamp)
 
 
-def render(message, template_name, context={}):
+def render(message, template_name, context=None):
+    context = context or dict()
     base_dir = os.path.join(os.getcwd(), 'templates')
     paths = [base_dir]
     # Include paths on settings
-    # paths.extend(settings.TEMPLATES)
+    paths.extend(message.templates_dirs)
```

Comment:
My only comment is that we should get the template path directly from settings.

Reply:
I agree and I've tryied a couple of ways to perform it in a healthy way. 
I've acrhieved it by inspecting the stack and getting the Bottery instance (like falsk does), but it didn't worth it because I'd have to create a series of modules to get it right.

I.e.: Flask uses https://github.com/pallets/werkzeug/blob/master/werkzeug/local.py#L89 to access the app's settings.

Reply:
Actually, the Django way seens a bit more straightforward. I'll try to make a settings module based in django's pattern